test/test_cmake.py

```diff
@@ -27,6 +27,9 @@
 from hpccm.building_blocks.cmake import cmake
 
 class Test_cmake(unittest.TestCase):
+    def setUp(self):
+        super(TestAssertEqual, self).setUp()
+        self.addTypeEqualityFunc(str, self.assertMultiLineEqual)
```

Comment:
Is this supported for Python 2.7?

Reply:
This was accidental.  The diff produced by the tests is unusable, so I spent ~30 min figuring out why this test was failing, and it turns out the `L` in Linux changed from `L` to `l` or vice-versa. I added this to debug it, and forgot to remove it.